### PR TITLE
add eslint-plugin-sonarjs and eslint-plugin-no-unsanitized

### PR DESCRIPTION
**Pull request by ahcai**

### CODE REVIEW

File: packages/eslint-config-twilio-react/package.json [hunk 1]
```diff
@@ -42,9 +42,11 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-no-only-tests": "^2.4.0",
+    "eslint-plugin-no-unsanitized": "^3.1.4",
     "eslint-plugin-prettier": "^3.3.0",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-sonarjs": "^0.5.0",
```
Comment (ghostling) [inline packages/eslint-config-twilio-react/package.json:49]: 👀 isn't this already in eslint-config-twilio below?

Comment (ahcai) [reply packages/eslint-config-twilio-react/package.json:49]: hmm ya you're right. @ktalebian do you rmb why we have some other plugins as deps here when they're already deps of `eslint-config-twilio` like `eslint-plugin-import` and `eslint-plugin-prettier`?

Comment (ahcai) [reply packages/eslint-config-twilio-react/package.json:49]: o actually i think it's a lerna thing. i just tried removing it but lerna complained that it should be a dev-dependency

Comment (ktalebian) [reply packages/eslint-config-twilio-react/package.json:49]: It is a known lerna bug - setting up symlink for deps of deps does not work properly.

Comment (ghostling) [reply packages/eslint-config-twilio-react/package.json:49]: 👍 ahh that must be annoying :(

**Review by ghostling — APPROVED** [review]
don't think my +1 counts here, but have it anyway :D

**Review by ktalebian — APPROVED** [review]

**Pull request merged**